net/vpn-policy-routing/Makefile

```diff
@@ -39,8 +39,8 @@ define Package/vpn-policy-routing/install
 	$(INSTALL_DIR) $(1)/etc/init.d $(1)/etc/config $(1)/etc/hotplug.d/firewall $(1)/etc/
 	$(INSTALL_BIN) ./files/vpn-policy-routing.init $(1)/etc/init.d/vpn-policy-routing
 	$(SED) "s|^\(PKG_VERSION\).*|\1='$(PKG_VERSION)-$(PKG_RELEASE)'|" $(1)/etc/init.d/vpn-policy-routing
-	$(INSTALL_CONF) ./files/vpn-policy-routing.conf $(1)/etc/config/vpn-policy-routing
```

Comment:
Do you need the this file? If not, then remove it from this repository as well.

Reply:
Oh, thank you, good catch, I'll push the fix!